library.wdl

```diff
@@ -68,14 +68,15 @@ workflow Library {
     call preprocess.GatkPreprocess as preprocessing {
             input:
                 bamFile = markDuplicates.outputBam,
-                outputBamPath = outputDir + "/" + sampleId + "-" + sampleId + ".markdup.bqsr.bam",
+                basePath = outputDir + "/" + sampleId + "-" + sampleId + ".markdup.bqsr",
```

Comment:
Why is `sampleId` mentioned twice here? The output will be `outputDir/sampleId-sampleId.markdup.bsqr`.